bin/sandbox

```diff
@@ -53,6 +53,7 @@ gem 'solidus', github: 'solidusio/solidus', branch: '$BRANCH'
 gem 'solidus_auth_devise', '>= 2.1.0'
 gem 'rails-i18n'
 gem 'solidus_i18n'
+gem 'solidus_paypal_commerce_platform'
```

Comment:
I don't think this _should_ be necessary, but I'll have to look into it. I think there's been some work to fix some issues that cropped up from it being the new default, but I don't remember the state of the specifically off the top of my head.

Can you make this PR just the extension name fix, and then I'll look into the paypal thing? (You can of course still keep using this fix locally.)

Reply:
You got it! Going to open a new PR because if I force push to this branch, the name of the branch will (misleadingly) still have `and_add_missing_dependency` in it.